app/assets/javascripts/selectizer.js.coffee

```diff
@@ -47,3 +47,24 @@ $ ->
     selectizer = new Selectizer(el)
     selectizer.enable()
     selectizer.add_label()
+  
+  $('#new_ncr_work_order').parsley 
```

Comment:
This'd be a bit better if it weren't specific to the work order page. Can you think of a way to implement the same thing more declaratively (e.g. the selectize and filtering data attributes)?